components/qr-code/style/index.ts

```diff
@@ -63,15 +64,16 @@ const genQRCodeStyle: GenerateStyle<QRCodeToken> = (token) => {
   };
 };
 
-export const prepareComponentToken: GetDefaultToken<'QRCode'> = () => ({});
+export const prepareComponentToken: GetDefaultToken<'QRCode'> = (token) => ({
+  QRCodeTextColor: new TinyColor(token.colorText).toRgbString(),
```

Comment:
这个不需要用 tinycolor 转换，还是写在下面
```suggestion
  QRCodeTextColor: token.colorText,
```